lib/sentry/event.ex

```diff
@@ -56,6 +58,8 @@ defmodule Sentry.Event do
 
     stacktrace = Keyword.get(opts, :stacktrace, [])
 
+    fingerprint = Keyword.get(opts, :fingerprint, ["{{ default }}"])
```

Comment:
I'm not sure if we should ALWAYS send this up, or if sentry will ignore it if its `fingerprint: nil` or `fingerprint: []`